.docker/prod.Dockerfile

```diff
@@ -36,6 +36,7 @@ COPY --chown=www-data:www-data --from=builder /var/www/vendor/ ./vendor/
 
 #  Declare the volumes
 VOLUME /var/www/public/images/profiles
+VOLUME /var/www/resources/breakages
```

Comment:
Do we need to add a volume mount to the `docker-compose.yaml` file (and update the `local.Dockerfile` file) so that this directory is also kept for local dev?

Reply:
It should persist because the whole of `/var/www` is a volume in `local.Dockerfile`